ansible_collections/arista/avd/molecule/eos_cli_config_gen/intended/configs/port-channel-interfaces.cfg

```diff
@@ -165,6 +165,25 @@ interface Port-Channel99
 interface Port-Channel100
    logging event link-status
    no switchport
+   switchport tap native vlan 10
+   switchport tap identity 3
+   switchport tap mac-address dest 01:00:00:00:00:00 src 01:23:45:67:89:ab
+   switchport tap encapsulation gre strip
+   switchport tap encapsulation gre destination 1.1.1.1 protocol 0x0000 strip
+   switchport tool mpls pop all
+   switchport tool encapsulation vn-tag strip
+   switchport tool encapsulation dot1br strip
+   switchport tap allowed vlan 25
+   switchport tool allowed vlan 23
+   switchport tool identity qinq
+   switchport tool identity qinq source dzgre port inner policy
+   switchport tap truncation
+   switchport tap default group g1 group g2 group g3
+   switchport tap default nexthop-group nexthop_g1 nexthop_g2 nexthop_g3
+   switchport tap default interface ethernet4
+   switchport tap default interface port-channel10
+   switchport tool group set group1 group2 group3
+   switchport tool dzgre preserve
```

Comment:
This is invalid command for me, can you recheck
```
wan1-dc1(config-if-Et1)#switchport tool dzgre preserve
% Invalid input
```

Reply:
its working for me-
```
s1-leaf2(config-if-Et1)#switchport tool dzgre preserve
s1-leaf2(config-if-Et1)#show active
interface Ethernet1
   description MLAG_PEER_s1-leaf1_Ethernet1
   channel-group 1 mode active
   lldp tlv transmit ztp vlan 10
   switchport tap identity 10
   switchport tap mac-address dest 00:01:00:01:00:01
   switchport tool dzgre preserve
```